Description: Ignores the tests using geronimo-transaction which isn't available in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -272,6 +272,20 @@
             <source>1.5</source>
             <target>1.5</target>
           </configuration>
+          <executions>
+            <execution>
+              <id>default-testCompile</id>
+              <phase>test-compile</phase>
+              <goals>
+                <goal>testCompile</goal>
+              </goals>
+              <configuration>
+                <testExcludes>
+                  <exclude>**/managed/**</exclude>
+                </testExcludes>
+              </configuration> 
+            </execution>
+          </executions>
         </plugin>
       </plugins>
     </build>
